var/spack/repos/builtin/packages/cbtf-argonavis/package.py

```diff
@@ -43,16 +43,16 @@
 from spack import *
 
 
-class CbtfArgonavis(Package):
+class CbtfArgonavis(CMakePackage):
     """CBTF Argo Navis project contains the CUDA collector and supporting
        libraries that was done as a result of a DOE SBIR grant.
 
     """
     homepage = "http://sourceforge.net/p/cbtf/wiki/Home/"
 
     # Mirror access template example
-    # url = "file:/home/jeg/OpenSpeedShop_ROOT/SOURCES/cbtf-argonavis-1.6.tar.gz"
-    # version('1.6', '0fafa0008478405c2c2319450f174ed4')
+    # url = "file:/home/jeg/cbtf-argonavis-1.8.1.tar.gz"
+    # version('1.8.1', 'b63db444fff92370a88197882a8d54d0')
```

Comment:
I don't think this mirror URL is a good suggestion. The better way is to use a tradition Spack mirror: http://spack.readthedocs.io/en/latest/mirrors.html. This doesn't require you to make any modifications to the package.